src/cmd/initialize.go

```diff
@@ -28,16 +30,53 @@ var initCmd = &cobra.Command{
 		initPackageName := fmt.Sprintf("zarf-init-%s.tar.zst", config.GetArch())
 		config.DeployOptions.PackagePath = initPackageName
 
-		// Use an init-package in the executable directory if none exist in current working directory
+		// Try to use an init-package in the executable directory if none exist in current working directory
 		if utils.InvalidPath(config.DeployOptions.PackagePath) {
 			executablePath, err := utils.GetFinalExecutablePath()
 			if err != nil {
-				message.Fatal(err, "Unable to get the directory where the zarf cli executable lives")
+				message.Error(err, "Unable to get the directory where the zarf cli executable lives")
 			}
 
 			executableDir := path.Dir(executablePath)
 			config.DeployOptions.PackagePath = filepath.Join(executableDir, initPackageName)
+
+			// If the init-package doesn't exist in the executable directory, suggest trying to download
+			if utils.InvalidPath(config.DeployOptions.PackagePath) {
+
+				// If no CLI version exists (should only occur in dev or CI), try to get the latest release tag from Githhub
+				if _, err := semver.StrictNewVersion(config.CLIVersion); err != nil {
+					config.CLIVersion, err = utils.GetLatestReleaseTag(config.GithubProject)
+					if err != nil {
+						message.Fatal(err, "No CLI version found and unable to get the latest release tag for the zarf cli.")
+					}
+				}
+
+				confirmDownload := config.DeployOptions.Confirm
+				url := fmt.Sprintf("https://github.com/%s/releases/download/%s/%s", config.GithubProject, config.CLIVersion, initPackageName)
+
+				// Give the user the choice to download the init-package and note that this does require an internet connection
+				message.Question("It seems the init package could not be found locally, Zarf can download this for you if you still have internet connectivity.")
+				message.Question(url)
+
+				// Prompt the user if --confirm not specified
```

Comment:
An extra flag for this would also have the added benefit of skirting around the extra downloads in the automated testing from your other comment.

Reply:
An on a UX experience from the air gap world, since we're not validating internet connectivity to GitHub this would probably throw an error or a timeout on the `utils.DownloadToFile( ... )` call, making it appear to the user as a Zarf bug instead of a user-configuration issue. 

It's a small perspective switch but might slow them down in figuring out whats wrong and slowing down their path to mission.

Reply:
This is a good point on downloading remote files without prompting, I think that this really is targeting the de user/testing online anyway where you wouldn't normally see `--confirm` unless a dev is doing it for rapid development.  So instead I changed it to just not support init package download if not explicitly provided via the cli.  
<img width="793" alt="Screen Shot 2022-06-08 at 4 22 11 PM" src="https://user-images.githubusercontent.com/882485/172720078-1d66a4e0-d588-4496-99ee-9241faf29b7e.png">


Added an explanation regarding your last point. 
<img width="746" alt="Screen Shot 2022-06-08 at 4 15 14 PM" src="https://user-images.githubusercontent.com/882485/172718671-65c53148-2562-4fba-b89b-dabff9d191a9.png">